src/app/main/table/01.basic/table-basic.component.ts

```diff
@@ -63,6 +66,7 @@ export class TableBasicComponent {
       filter: this.quickFilterToggle.checked,
       filterCaseSensitive: this.filterCaseSensitive.checked,
       exportButtonToggle: this.exportButtonToggle.checked,
+      configurationButtonToggle: this.configurationButtonToggle.checked,
```

Comment:
Use this value in table-utils in order to display the attribute value in code example.